From 5800d1d1565a5fc61349faa9e52bd1ca6be8fe43 Mon Sep 17 00:00:00 2001
From: Tim Collier <osdevtc@gmail.com>
Date: Fri, 25 Oct 2019 09:41:26 +0100
Subject: [PATCH] staging: wlan-ng: remove unused field from struct hfa384x_usbctlx
Git-commit: 5800d1d1565a5fc61349faa9e52bd1ca6be8fe43
Patch-mainline: v5.5-rc1
References: jsc#SLE-13430

The variant field in struct hfa384x_usbctlx is not referenced anywhere
in the driver, so remove it.

Signed-off-by: Tim Collier <osdevtc@gmail.com>
Link: https://lore.kernel.org/r/20191025084126.9181-6-osdevtc@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/staging/wlan-ng/hfa384x.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/wlan-ng/hfa384x.h b/drivers/staging/wlan-ng/hfa384x.h
index 86fd3fcd8a36..bdd7f414fdbb 100644
--- a/drivers/staging/wlan-ng/hfa384x.h
+++ b/drivers/staging/wlan-ng/hfa384x.h
@@ -1181,8 +1181,6 @@ struct hfa384x_usbctlx {
 	ctlx_cmdcb_t cmdcb;	/* Async command callback */
 	ctlx_usercb_t usercb;	/* Async user callback, */
 	void *usercb_data;	/*  at CTLX completion  */
-
-	int variant;		/* Identifies cmd variant */
 };
 
 struct hfa384x_usbctlxq {
-- 
2.16.4

